tests/conftest.py

```diff
@@ -26,6 +28,16 @@ def clear_all_traces():
     test_exporter.clear()
 
 
+@pytest.fixture(autouse=True)
+def clear_audit_log(request):
+    # only if we may have used the db
+    for mark in request.node.own_markers:
+        if mark.name == "django_db":
+            # truncate should be much faster than AuditLog.objects.all().delete()
```

Comment:
I know what you mean, but talking about truncating and then not calling `TRUNCATE ...` is a little bit jarring.

Reply:
Ah, that's how you spell TRUNCATE in sqlite.

Reply:
Have removed this commit, as in hind sight it was doing nothing.